fvm/Cargo.toml

```diff
@@ -31,7 +31,7 @@ lazy_static = "1.4.0"
 derive-getters = "0.2.0"
 derive_more = "0.99.17"
 replace_with = "0.1.7"
-filecoin-proofs-api = { version = "11", default-features = false }
+filecoin-proofs-api = { version = "11.1", default-features = false }
```

Comment:
rust noob here  - confirm we dont need the 11.1`.0` ? https://github.com/filecoin-project/rust-filecoin-proofs-api/releases/tag/v11.1.0

Reply:
Question for @Stebalien, but I'm guessing the answer is no since it builds.